corehq/apps/sso/views/saml.py

```diff
@@ -92,6 +98,7 @@ def sso_saml_acs(request, idp_slug):
         "processed_response": processed_response,
         "saml_relay": saml_relay,
         "request_session_data": request_session_data,
+        "login_error": getattr(request, 'sso_login_error', None),
```

Comment:
Is this response for debugging purposes? If not, are we exposing any potentially sensitive information here? (Moreso with the other properties here like `request_session_data` -- `sso_login_error` doesn't expose anything)

Reply:
yeah this whole response is debug only. there will be a final redirect to a page with more user-friendly responses